{"primaryContentSections":[{"kind":"declarations","declarations":[{"languages":["swift"],"tokens":[{"text":"let","kind":"keyword"},{"text":" ","kind":"text"},{"text":"reason","kind":"identifier"},{"text":": ","kind":"text"},{"text":"String","preciseIdentifier":"s:SS","kind":"typeIdentifier"}],"platforms":["Linux"]}]}],"variants":[{"paths":["\/documentation\/openapikit\/validationerror\/reason"],"traits":[{"interfaceLanguage":"swift"}]}],"hierarchy":{"paths":[["doc:\/\/OpenAPIKit\/documentation\/OpenAPIKit","doc:\/\/OpenAPIKit\/documentation\/OpenAPIKit\/ValidationError"]]},"kind":"symbol","abstract":[{"text":"The reason for the validation failure.","type":"text"}],"schemaVersion":{"major":0,"patch":0,"minor":3},"identifier":{"url":"doc:\/\/OpenAPIKit\/documentation\/OpenAPIKit\/ValidationError\/reason","interfaceLanguage":"swift"},"metadata":{"modules":[{"name":"OpenAPIKit"}],"symbolKind":"property","role":"symbol","title":"reason","fragments":[{"kind":"keyword","text":"let"},{"kind":"text","text":" "},{"kind":"identifier","text":"reason"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"String","preciseIdentifier":"s:SS"}],"externalID":"s:10OpenAPIKit15ValidationErrorV6reasonSSvp","roleHeading":"Instance Property"},"sections":[],"references":{"doc://OpenAPIKit/documentation/OpenAPIKit/ValidationError/reason":{"url":"\/documentation\/openapikit\/validationerror\/reason","identifier":"doc:\/\/OpenAPIKit\/documentation\/OpenAPIKit\/ValidationError\/reason","type":"topic","kind":"symbol","role":"symbol","title":"reason","fragments":[{"text":"let","kind":"keyword"},{"text":" ","kind":"text"},{"text":"reason","kind":"identifier"},{"text":": ","kind":"text"},{"text":"String","preciseIdentifier":"s:SS","kind":"typeIdentifier"}],"abstract":[{"text":"The reason for the validation failure.","type":"text"}]},"doc://OpenAPIKit/documentation/OpenAPIKit/ValidationError":{"kind":"symbol","navigatorTitle":[{"text":"ValidationError","kind":"identifier"}],"abstract":[{"text":"Validation errors are just a textual reason for validation failure and","type":"text"},{"text":" ","type":"text"},{"text":"a coding path where the validation error occurred.","type":"text"}],"type":"topic","identifier":"doc:\/\/OpenAPIKit\/documentation\/OpenAPIKit\/ValidationError","role":"symbol","title":"ValidationError","fragments":[{"text":"struct","kind":"keyword"},{"text":" ","kind":"text"},{"text":"ValidationError","kind":"identifier"}],"url":"\/documentation\/openapikit\/validationerror"},"doc://OpenAPIKit/documentation/OpenAPIKit":{"role":"collection","title":"OpenAPIKit","abstract":[],"kind":"symbol","identifier":"doc:\/\/OpenAPIKit\/documentation\/OpenAPIKit","url":"\/documentation\/openapikit","type":"topic"}}}